ckanext/harvester_dashboard/controllers.py

```diff
@@ -2,16 +2,140 @@
 
 import logging
 from ckan.lib.base import BaseController, render
-from ckan.common import c
+from ckan.common import c, request, _
 import ckan.plugins.toolkit as tk
+from ckanext.harvester_dashboard.helpers import \
+    harvester_dashboard_organization_title
 log = logging.getLogger(__name__)
+RESULT_GATHER_ERRORS = 'gather-errors'
+RESULT_IMPORT_ERRORS = 'import-errors'
+RESULT_NO_UPDATES = 'no-updates'
+RESULT_OK = 'ok'
+RESULT_ALL = 'all'
+RESULT_OPTIONS = [{'text': _('Results: all results'),
+                   'value': RESULT_ALL},
+                  {'text': _('Results: gather errors'),
+                   'value': RESULT_GATHER_ERRORS},
+                  {'text': _('Results: import errors'),
+                   'value': RESULT_IMPORT_ERRORS},
+                  {'text': _('Results: no updates'),
+                   'value': RESULT_NO_UPDATES},
+                  {'text': _('Results: okay'),
+                   'value': RESULT_OK}]
+RUN_OPTIONS = [{'text': _('Run: all'), 'value': 'all'},
```

Comment:
As we have `RESULT_ALL` defined as a constant, can we use it here and in the other places where we're passing in `'all'` as a string? (Maybe it should be renamed in that case.)